src/test/dummyFunction.js

```diff
@@ -0,0 +1,3 @@
+export  const sum =(a, b)=> {
+ return a + b;
+}
```

Comment:
you don't need to create this file for this chore. Try to avoid creating files that would not be needed later on during the development cycle of the project.

Reply:
I will remove that then. Thanks for this